geometry_msgs/Pose2D goal
geometry_msgs/Pose2D start
int8_image map
int8_image[] cost_maps
int64 iterations
int64 segment_length
int64 number_of_trees
int64 objective_number
bool minimum_distance_enabled
